samples/outgoing-webhook/csharp/WebhookSampleBot/Web.config

```diff
@@ -4,12 +4,15 @@
   http://go.microsoft.com/fwlink/?LinkId=301879
   -->
 <configuration>
-  <appSettings></appSettings>
+  <appSettings>
+    <add key="MS_OutgoingWebHookSecretToken" value="uDGKhpdsY6eSyIRdYEueTfeO2ftlMOEmi9+eJ7RaocM=" />
```

Comment:
Please remove the value from config file. If this is unused then please remove it.

Reply:
Removed value from web.config